### DISCUSSION TITLE
Discrepancies between TNS and CHIME/FRB catalogs?

### DISCUSSION THREAD

**Discussion by thandal** [chime-frb-open-data/community#86] [Q&A]
Hi,
I've been trying to assemble a large catalog of FRBs. I pulled data from both TNS and the CHIME/FRB catalog 1, and then compared some basic fields (ra, dec, dm, time).  TNS has 837 FRB events, CHIME/FRB catalog 1 has 600 events, and they have 536 events in common. I was expecting a certain amount of rounding-grade error, and maybe even some systematic differences (for example in time standards). But I found a lot more error than I expected, in all the fields!

![image](https://user-images.githubusercontent.com/2613832/229863027-c80bb650-6943-4a97-b276-6651888c116a.png)

Colab notebook for the comparison: https://colab.research.google.com/drive/1SXL9emCrTKitsRLmg-Z_5ijQKrDU97Up

Would you have any guidance on this? My inclination is to trust the CHIME/FRB data, but leaves me suspicious of the data quality of the other ~240 events in TNS.

Thanks,

**Comment by danielemichilli** [comment]
Hi, CHIME/FRB Catalog 1 only has 536 events. You are probably seeing 600 because the properties of single sub-bursts were reported separately for multi-component bursts. If you consider unique TNS names, they are 536
